README.md

```diff
@@ -94,6 +94,20 @@ Create a new node.js configuration, add `-r ts-node/register` to node args and m
 
 **Note:** If you are using the `--project <tsconfig.json>` command line argument as per the [Configuration Options](#configuration-options), and want to apply this same behavior when launching in VS Code, add an "env" key into the launch configuration: `"env": { "TS_NODE_PROJECT": "<tsconfig.json>" }`.
 
+### Intellij (and Webstorm)
+
+Run > Edit Configurations > click the plus icon at the top left > Node.js
+
+Node interpreter: node.js (the default should be fine)
+Node parameters: `--inspect=0.0.0.0:9229 --require ts-node/register --require tsconfig-paths/register`
```

Comment:
We don't mention tsconfig-paths/register in the VSCode example, so this is slightly inconsistent.  I would omit the extra --require and limit it to `ts-node/register`

Reply:
Is binding to 0.0.0.0 ok?  That'll allow external machines to connect to the process, right?

Reply:
> Is binding to 0.0.0.0 ok? That'll allow external machines to connect to the process, right?

yes, i think it's because we sometimes run it inside a docker instance